addons/web/static/src/core/network/rpc_cache.js

```diff
@@ -120,10 +122,15 @@ export class RPCCache {
      * @param {function} fallback
      * @param {RPCCacheSettings} settings
      */
-    read(table, key, fallback, { callback = () => {}, type = "ram", update = "once" } = {}) {
+    read(table, key, fallback, { callback = () => {}, type = "ram", update = "once", maxAge = ONE_YEAR } = {}) {
```

Comment:
could you lint this file?

You can do it automatically by enabling the web tooling (`./addons/web/tooling/enable.sh`), which will basically enable a pre-commit hook that'll lint all altered files.